### PR TITLE
2 patmos x

### LINKED ISSUE #2: PATMOS-x
- [x] Identify which data is need
- [x] Write download script
- [ ] Pre-process data

### PR DESCRIPTION
**Pull request by adriaat**
- Script to download relevant PATMOS-x data
- Add packages to environment

### CODE REVIEW

**Comment by adriaat** [comment]
The variables of interest are hard-coded, for now.

File: scripts/PATMOS-x_download.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+        )
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--start",
+        required=True,
+        type=lambda x: datetime.datetime.strptime(x, "%Y%m%d"),
+        help="from which date (YYYMMDD) download PATMOS-x data",
```
Comment (JuliaKukulies) [inline scripts/PATMOS-x_download.py]:
```suggestion
        help="from which date (YYYYMMDD) download PATMOS-x data",
```

File: scripts/PATMOS-x_download.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    )
+    parser.add_argument(
+        "--end",
+        required=True,
+        type=lambda x: datetime.datetime.strptime(x, "%Y%m%d"),
+        help="until which date (YYYMMDD) download PATMOS-x data",
```
Comment (JuliaKukulies) [inline scripts/PATMOS-x_download.py]:
```suggestion
        help="until which date (YYYYMMDD) download PATMOS-x data",
```

**Review by JuliaKukulies — APPROVED** [review]
Thanks for adding your download script, @adriaat! I tested installing the updated conda environment and downloading files with your script and everything works fine :)

**Pull request merged**